resources/js/Pages/Results.vue

```diff
@@ -1,28 +1,50 @@
 <template>
     <div class="page-container results-page">
         <Head title="Mismatch Finder - Results" />
-        <section id="results" v-if="Object.keys(results).length">
-            <section class="item-mismatches" v-for="(mismatches, item, idx) in results" :key="idx">
+        <div id="top-button">
+                <wikit-button
+                    variant="normal"
+                    native-type="button"
+                    type="neutral"
+                    @click.native="returnHome"
+                >
+                    &#x2190; {{ $i18n('refine-items-selection') }}
+                </wikit-button>
+            </div>
+            <section id="description-section">
+                <h2 class="h4">{{ $i18n('results-page-title') }}</h2>
+                <p id="about-description" >{{ $i18n('results-page-description') }}</p>
+            </section>
+        <div v-if="notFoundItemIds.length" class="not-found">
```

Comment:
I think the `<div>` isn't needed here, as the `v-if` can go into the `<Message>` element, which will also keep the section structure intact, when the Message is not displayed. See [@itamargiv's comment](https://github.com/wmde/wikidata-mismatch-finder/pull/110#discussion_r713971650) on the Homepage error message.

Reply:
I would actually put it on the `section` element, so that it doesn't appear on the page at all if there are no messages